packages/@dcl/react-ecs/src/components/uiTransform/types.ts

```diff
@@ -124,4 +126,6 @@ export interface UiTransformProps {
   flexShrink?: number
   /** The overflow property controls what happens to content that is too big to fit into an area */
   overflow?: OverflowType
+  /** The pointer filter mode that specifies if the element should block the pointer or not (elements with pointer events always block the pointer regardless of this property) **/
+  pointerFilter?: PointerFilterMode | undefined
```

Comment:
LGTM but maybe we can create a helper to avoid using the enums here, like we do in the overflow line above.
instead of `pointerFilter: PointerFilterMode.PFM_Block` do something like, `pointerFioter: 'block' | 'none'`

Reply:
Cool, sounds good to me, let me give it a try.

Reply:
Done!